Test/FMI3/fmu_dummy/fmu3_model.c

```diff
@@ -36,10 +36,10 @@
 /* Model calculation functions */
 static int calc_initialize(instance_ptr_t inst)
 {
-    inst->states[VAR_R_HEIGHT]        = 1.0;
+    inst->states[VAR_R_HEIGHT]       = 1.0;
     inst->states[VAR_R_HEIGHT_SPEED] = 4;
-    inst->reals    [VAR_R_GRATIVY]        = -9.81;
-    inst->reals    [VAR_R_BOUNCE_CONF]    = 0.5;
+    inst->reals[VAR_R_GRATIVY]       = -9.81;
```

Comment:
I know this was the case even before but maybe we can change `GRATIVY` to `GRAVITY`